### ISSUE TITLE
'Leave' transition still not displaying

### ISSUE THREAD

**Issue by tinymachine** [LinusBorg/portal-vue#153]
(Opening a new issue to call attention to issue #98, which was closed with commit #145, but which still doesn't seem to be fixed, as far as I can tell.)

There still appears to be an issue with 'portaled' modals not displaying their 'leave' transition. Demos:

- [portal-vue 1.3.0 demo](https://vue-portal-transition-leave-issue.netlify.com)
- [portal-vue 1.4.0 demo](https://portal-vue-1-4-0--vue-portal-transition-leave-issue.netlify.com)

Links to the repos are provided on the demo pages. I'd really appreciate any help anyone can provide. Thanks to @LinusBorg for creating and sharing such a useful plugin!

**Comment by LinusBorg** [comment]
You're right, in this scenario it still doesn't work - and I have no idea why.

Might take a while to debug this, if at all.

**Comment by LinusBorg** [comment]
I'll look into it.

**Comment by tinymachine** [comment]
Thank you!

**Comment by LinusBorg** [comment]
@tinymachine can you share the source for these demos?

**Comment by tinymachine** [comment]
Sure, @LinusBorg: 

1.3.0 demo: https://github.com/tinymachine/portal-vue-transition-examples/tree/master
1.4.0 demo: https://github.com/tinymachine/portal-vue-transition-examples/tree/portal-vue-1.4.0

**Comment by LinusBorg** [comment]
Thanks

**Comment by tinymachine** [comment]
Thanks for looking into this issue, or even thinking about looking into it!

**Comment by albertpratomo** [comment]
Any updates on this issue?

**Comment by shumuko** [comment]
@tinymachine I just tried your 1.4.0 demo: https://github.com/tinymachine/portal-vue-transition-examples/tree/portal-vue-1.4.0 demo. Seems the 'leave' animation is displaying fine.

**Comment by tinymachine** [comment]
@shumuko Are you sure? When I look at the [1.4.0 demo](https://portal-vue-1-4-0--vue-portal-transition-leave-issue.netlify.com) linked to in [my initial comment](https://github.com/LinusBorg/portal-vue/issues/153#issue-368876014), I'm still not seeing the out-transition when you click "Launch Modal via Portal" and then click the Close button. The modal closes abruptly, without fading out the overlay and doing the transition on the modal.

**Comment by shumuko** [comment]
@tinymachine I downloaded your files, ran `yarn` and  the server locally. It did work.

**Comment by tinymachine** [comment]
@shumuko, that's great that it's working. It looks like in my repo I didn't pin `portal-vue` to version 1.4.0 as I should have (see the [package.json](https://github.com/tinymachine/portal-vue-transition-examples/blob/ac1d985b853fb5783f4c9c8e0e84242df6a8cb67/package.json#L11)) since that was the current version when I created this issue. This might account for why it's working for you. Perhaps a newer version was installed when you ran `yarn` and this issue was addressed in a subsequent version? 

I'll close this issue as it sounds like it has been addressed. Cheers!
